src/components/Editor/SearchBar.js

```diff
@@ -148,13 +149,14 @@ class SearchBar extends Component<Props, State> {
   toggleSearch = (e: SyntheticKeyboardEvent<HTMLElement>) => {
     e.stopPropagation();
     e.preventDefault();
-    const { editor } = this.props;
+    const { editor, closeQuickOpen, searchOn, setActiveSearch } = this.props;
 
-    if (!this.props.searchOn) {
-      this.props.setActiveSearch("file");
+    if (!searchOn) {
+      closeQuickOpen();
+      setActiveSearch("file");
```

Comment:
can we have `setActiveSearch` close quick open if it's open? that will simplify the component

Reply:
Sure.